tests/fluka/test_fluka_converter.py

```diff
@@ -63,26 +62,19 @@
 STOP
 """
 
-_Test_dir = "./test_runs"
-
 @pytest.fixture
-def parser() -> FlukaParser:
+def parser() -> Parser:
     """Parser fixture."""
     return get_parser_from_str("fluka")
 
-@pytest.fixture
-def output_dir(tmp_path_factory) -> str:
-    """Fixture that creates a temporary dir for testing converter output."""
-    output_dir = tmp_path_factory.mktemp(_Test_dir)
-    return output_dir
 
 def test_parser(parser) -> None:
     """Check if parser is created correctly."""
     assert parser.info["version"] == "unknown"
     assert parser.info["simulator"] == "fluka"
```

Comment:
There is no check for `label` value.

```suggestion
    assert parser.info["simulator"] == "fluka"
    assert parser.info["label"] == ""
```